com.unity.netcode.gameobjects/CHANGELOG.md

```diff
@@ -10,9 +10,12 @@ Additional documentation and release notes are available at [Multiplayer Documen
 
 ### Added
 
+- Added `NetworkVariableWritePermission` to `NetworkVariableBase` and implemented `Owner` writable netvars. (#1762)
```

Comment:
dunno if we want to mention this adds client driven capabilities? Just so users do the association?